packages/rollup-plugin-html/src/output/adjustToFullAbsoluteUrls.ts

```diff
@@ -0,0 +1,47 @@
+import { Node, Document } from 'parse5';
+import { findElements, getTagName, getAttribute, setAttribute } from '@web/parse5-utils';
+
+function isAbsoluteableNode(node: Node) {
+  const metaAttributes = ['og:url', 'og:image'];
+  switch (getTagName(node)) {
+    case 'link':
+      if (getAttribute(node, 'rel') === 'canonical' && getAttribute(node, 'href')) {
+        return true;
+      }
+      return false;
+    case 'meta':
+      if (
+        metaAttributes.includes(getAttribute(node, 'property')!) &&
+        getAttribute(node, 'content')
+      ) {
+        return true;
+      }
+      return false;
+    default:
+      return false;
+  }
+}
+
+function joinSiteUrlAndPart(siteUrl: string, part: string) {
```

Comment:
I think you can use `new URL(url, pathOrUrl).href`

Reply:
thx worked 👍